_sass/vendors/bourbon/css3/_linear-gradient.scss

```diff
@@ -24,7 +24,7 @@
 
   $full: $g1, $g2, $g3, $g4, $g5, $g6, $g7, $g8, $g9, $g10;
 
-  // Set $g1 as the default fallback color
```

Comment:
It's possible this is a vendored file and should be ignored. I didn't look carefully. Sometimes I'll make PRs to upstreams, but atm I have a huge backlog and am already going down too many rabbit holes to be able to do that.